stage1_fly/aci/aci-manifest.in

```diff
@@ -28,6 +28,14 @@
         {
             "name": "coreos.com/rkt/stage1/gc",
             "value": "/gc"
+        },
+        {
+            "name": "coreos.com/rkt/stage1/stop",
+            "value": "/stop_fly"
+        },
+        {
+            "name": "coreos.com/rkt/stage1/interface-version",
+            "value": "3"
```

Comment:
So maybe we shouldn't add entry with interface version and live it like it was? Is there any purpose for this PR to add it? @iaguis